region/maridia/inner-yellow/Plasma Room.json

```diff
@@ -228,18 +228,79 @@
     {
       "id": 7,
       "link": [1, 1],
-      "name": "Space Pirate Farm",
+      "name": "Pirate Farm (All Pirates)",
       "requires": [
-        "SpaceJump",
+        {"resetRoom": {"nodes": [1]}},
+        {"or": [
+          {"and": [
+            "HiJump",
+            "SpaceJump",
+            {"cycleFrames": 930}
+          ]},
+          {"and": [
+            "HiJump",
+            "canTrickyWalljump",
+            {"cycleFrames": 960}
+          ]},
+          {"and": [
+            "SpaceJump",
+            {"cycleFrames": 1220}
+          ]},
+          {"and": [
+            "canSpringwall",
+            {"cycleFrames": 1035},
+            "h_doubleEquipmentScreenCycleFrames"
+          ]},
+          {"and": [
+            "HiJump",
+            "canSpringBallJumpMidAir",
+            {"cycleFrames": 910},
+            "h_doubleEquipmentScreenCycleFrames"
+          ]},
+          {"and": [
+            "canDoubleBombJump",
+            "canWalljump",
```

Comment:
```suggestion
            "canWallJumpBombBoost",
```

Reply:
Nice, didn't realize the crouch jump worked there. Added that as an option now. I figure it's ok if there's some extra lenience for the double bomb jump variants since it's probably expected to fall down sometimes. And it seemed too complicated to look at the interaction between weapon and movement. But feel free to make code suggestions or a follow-up PR if there's stuff you want to add/change here.